### PR TITLE
1766 partners desktop page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- Add search functionality to `/partners/desktop` page

## QA
- Run the website locally using `./run`
- Go to [/partners/desktop](http://0.0.0.0:8002/partners/desktop) and search for partners and filter partners 

## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1766

### CODE REVIEW

File: templates/partners/desktop.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-desktop">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Can you use just one class for this and add `color: #fff` to the suru styling?

For example:
``` html
<section class="p-strip--suru-desktop">
  <div class="row">
    ...
  </div>
</section>
```

File: templates/partners/desktop.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    </div>
+    <div class="row">
+      <div class="col-8">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+        </ul>
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: This is a lot of Lorem Ipsum here. Should we land pages with this content?

File: templates/partners/desktop.html [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+      </div>
+      <div class="col-4 u-align--center u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/eb9e7ede-What-youget-working-withus-icon.svg" alt="">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: I think it looks better if this image is centered including the title of the section. Can you move the title into the same row as this content please

File: templates/partners/desktop.html [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-6 u-align--center u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/73c9b800-increase+sales+icon.svg" alt="">
+      </div>
+      <div class="col-6">
+        <h2>Increase your sales by becoming Ubuntu Certified</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Rerum veniam eveniet at nam odio culpa voluptate labore. Sequi vero quo dolore, corporis a eaque id exercitationem qui minus, repudiandae consectetur.</p>
+        <!-- Add url -->
+        <a href="">Learn more about certification&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: This link should probably go to the certification site: https://certification.ubuntu.com/

File: templates/partners/desktop.html (hunk 2 above)
Comment (solazio) [reply templates/partners/desktop.html]: Don't know...we can hold it off if you think is better.

Comment (solazio) [reply templates/partners/desktop.html]: @anthonydillon let's land this page, as I have it captured in the spreadsheet that it still needs some work.

Comment (anthonydillon) [reply templates/partners/desktop.html]: Yea ok. Let's hold off updating the copy until it's signed off to save wasting time.

File: templates/partners/desktop.html [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+  </section>
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-6 u-align--center u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/73c9b800-increase+sales+icon.svg" alt="">
+      </div>
+      <div class="col-6">
+        <h2>Increase your sales by becoming Ubuntu Certified</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Rerum veniam eveniet at nam odio culpa voluptate labore. Sequi vero quo dolore, corporis a eaque id exercitationem qui minus, repudiandae consectetur.</p>
+        <a href="https://certification.ubuntu.com/">Learn more about certification&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: External link

File: templates/partners/desktop.html [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+{% block content %}
+  <section class="p-strip--suru-desktop">
+    <div class="row">
+      <div class="col-6">
+        <h1>Desktop</h1>
+        <p>Partnering with Canonical is a great way to keep your products ahead of the competition. The Technical Partner Programme gives you access to information, tools and training that can help you get to market quicker and for less. And in retail and online, we work with our hardware partners on the promotion of Ubuntu to consumers and businesses.</p>
+      </div>
+      <div class="col-6 u-align--center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/690d0479-disco-dingo-laptop.png" alt="charmed kubernetes image">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: If you add alt text it should describe the image for users that cannot see it.

File: templates/partners/desktop.html [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/292d1694-become-a-partner-footer.svg" class="p-media-object__image">
+          <div class="p-media-object__details">
+            <h1 class="p-media-object__title">Become a partner</h1>
+            <p class="p-media-object__content u-sv3">Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
+            <a href="/partners/find-a-partner">Talk to us about becoming a partner&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Shouldn't this link be to become a partner?

File: templates/partners/desktop.html [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+          <div class="p-media-object__details">
+            <h1 class="p-media-object__title">Find a partner</h1>
+            <p class="p-media-object__content u-sv3">Looking for a supplier with Ubuntu expertise? Select from the complete list of our partners.</p>
+            <a href="/partners/find-a-partner">Find a partner&nbsp;&rsaquo;</a>
+          </div>
+        </div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/292d1694-become-a-partner-footer.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Alt

File: templates/partners/desktop.html [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+      <p>At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque.</p>
+      <a href="/partners/become-a-partner">Become a partner&nbsp;&rsaquo;</a>
+      <img class="u-hide--medium u-hide--small" src="https://assets.ubuntu.com/v1/4f3cd96e-Ubuntu_19.04__Disco_Dingo_.png" style="bottom: 0; max-height: 90%; position: absolute; right: 0; width: 35%;">
+    </div>
+  </section>
+  <section class="p-strip--mid-light is-deep">
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <div class="p-media-object">
+          <img src="https://assets.ubuntu.com/v1/2cdff5cc-magnifier-find-a-partner-footer.svg" class="p-media-object__image">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Alt

File: templates/partners/desktop.html [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+    </div>
+  </section>
+  <hr />
+  <section class="p-strip is-deep" style="position: relative;">
+    <div class="u-fixed-width">
+      <h2>Join the future of open source</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+      <p>At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque.</p>
+      <a href="/partners/become-a-partner">Become a partner&nbsp;&rsaquo;</a>
+      <img class="u-hide--medium u-hide--small" src="https://assets.ubuntu.com/v1/4f3cd96e-Ubuntu_19.04__Disco_Dingo_.png" style="bottom: 0; max-height: 90%; position: absolute; right: 0; width: 35%;">
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Alt

File: templates/partners/desktop.html [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+        </li>
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/c3176652-logo-emulex.jpg" width="230" alt="Emulex">
+        </li>
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/df4150e2-NEC-logo.svg" alt="NEC">
+        </li>
+      </ul>
+      <p class="u-align-text--center">
+        <a href="/partners/find-a-partner">See all Desktop partners&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/partners/desktop.html]: Can you filter with a query Param?

**Pull request merged**